src/textual/widgets/_key_panel.py

```diff
@@ -90,11 +90,14 @@ def render_description(binding: Binding) -> Text:
             get_key_display = self.app.get_key_display
             for multi_bindings in action_to_bindings.values():
                 binding, enabled, tooltip = multi_bindings[0]
-                key_display = " ".join(
-                    get_key_display(binding) for binding, _, _ in multi_bindings
-                )
+                unique_key_displays: list[str] = []
```

Comment:
I'd be tempted to go with this. Faster, but uses an idiom not everyone is familiar with.

```python
keys_display = " ".join(
    dict.fromkeys(
        get_key_display(binding) for binding, _, _ in multi_bindings
    )
)
```

Reply:
Thanks, I wasn't aware of `dict.fromkeys`.